woocommerce/woocommerce
# CYS: Track move up/move down pattern

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/48950 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Remove these code lines: https://github.com/woocommerce/woocommerce/blob/5d936bb547706a4fe1f2c24f7dda7a418d5fe814/packages/js/tracks/src/index.ts/#L52-L55
4. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`.
5. Click on `Design your homepage`.
6. Open the network tools.
7. Search `t.gif` in the network tools filter.
8. Move down the pattern.
9. Move up the pattern.
10. Ensure that `customize_your_store_assembler_pattern_move_up_click` and `customize_your_store_assembler_pattern_move_down_click` events are tracked.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

CYS: Track move up/move down pattern.

</details>
